FluxSharp.UI/Abstractions/Dispatcher.cs

```diff
@@ -1,53 +1,35 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace FluxSharp.Abstractions
 {
     public class Dispatcher
     {
-        readonly Dictionary<Type, List<object>> callbacks
-            = new Dictionary<Type, List<object>>();
+        private readonly IDictionary<Type, IList<object>> _callbacks =
+            new Dictionary<Type, IList<object>>();
 
         public void Register<T>(Action<T> callback)
         {
             var key = typeof(T);
 
-            if (callbacks.ContainsKey(key))
-            {
-                var currentCallbacks = callbacks[key];
-                currentCallbacks.Add(callback);
-            }
-            else
-            {
-                var newCallbacks = new List<object> { callback };
-                callbacks.Add(key, newCallbacks);
-            }
+            if(!_callbacks.ContainsKey(key))
+                _callbacks.Add(key, new List<object>());
+
+            _callbacks[key].Add(callback);
```

Comment:
The `else` statement should not be required. It is just making sure we have a kvp with an initialized list before we start adding callbacks to it.